build-wheelhouse/action.yml

```
@@ -89,7 +89,14 @@ runs:
     - name: "Verify library is properly installed and get its version number"
       shell: bash
       run: |
-        echo "library_version=$(python -c "from ${{ inputs.library-namespace }} import __version__; print(__version__)")" >> $GITHUB_ENV
+        library_version=$(python -c "from ${{ inputs.library-namespace }} import __version__; print(__version__)" || exit 1 )
```

```suggestion
        library_version=$(python -c "from ${{ inputs.library-namespace }} import __version__; print(); print(__version__)" | tail -1 || exit 1 )
```
@RobPasMue @jorgepiloto FYI This is what I had suggested in my wall of text here: https://github.com/ansys/actions/issues/269, to avoid failing due to extra unexpected stdout

To be clear, just a suggestion/food for thought, and I am not aware of any downsides to this change

---

Yep, I think we can add it as well. Should be okay.